src/meta/meta_bulk_load_service.cpp

```diff
@@ -17,12 +17,23 @@
 
 #include <dsn/dist/fmt_logging.h>
 #include <dsn/utility/fail_point.h>
+#include <dsn/utility/flags.h>
 
 #include "meta_bulk_load_service.h"
 
 namespace dsn {
 namespace replication {
 
+DSN_DEFINE_uint32("meta_server",
+                  bulk_load_max_failover_count,
+                  20,
```

Comment:
20 will still be too many if the job is pretty large and the disk will easily get burst out.
I would recommend 5 or 10.